### ISSUE TITLE
MOD_BASE_PATH not validated on install

### ISSUE THREAD

**Issue by gadamiak** [modxbot/migrate#4433]
gadamiak created Redmine issue ID 4433

@MODX_BASE_PATH@ is not validated during installation for existence of trailing slash. If entered without it installation fails to create @config.core.php@ in @MODX_BASE_PATH@ directory.

**Comment by opengeek** [comment]
opengeek submitted:

I don't understand where it is you mean it is not validated. Is this related to advanced distributions where you can change the locations? You mentioned this was using git though, so I'm confused as to how you would be setting MODX_BASE_PATH as this is automatically determined during installation.

**Comment by gadamiak** [comment]
gadamiak submitted:

I guess I was not clear enough. I made a fresh installation from git and adjusted paths for root, manager and connectors folders. While doing copy-and-paste I missed trailing slash in root path. This resulted in @MODX_BASE_PATH@ not having the trailing slash and @config.core.php@ not created. I had to manually edit it in my config file inside @core/config/@ folder to make the site work at all and copy @config.core.php@ from any of @manager@ or @connectors@ folders.

I meant the paths should be checked for the existence of trailing slash when modified during install. I hope it's clear now :)
